TypeCobol/Compiler/CodeModel/SymbolTable.cs

```diff
@@ -1184,7 +1181,12 @@ public List<Program> GetPrograms()
         /// <param name="symbol"></param>
         private void Add<T>([NotNull] IDictionary<string, List<T>> table, [NotNull] T symbol) where T : Node
         {
-            string key = symbol.QualifiedName.Head;
+            //QualifiedName of symbol can be null - if we have a filler in the type definition
+            if (symbol.QualifiedName == null)
+            {
+                return;
+            }
+            string key = symbol.QualifiedName.Head.ToLowerInvariant();
```

Comment:
Dictionaries in SymbolTable use StringComparer.OrdinalIgnoreCase for comparison.
Maybe it's faster to put the key in lower case and convert the searched value into lower case.
But with your code, the job is half done. You convert the key to lower case, but the Dictionary still use OrdinalIgnoreCase and the value searched is not converted in lower case.


Conversion to lower case is outside the scope of this issue, so please remove it from this PR.